pyproject.toml

```diff
@@ -91,7 +91,7 @@ scorched = {git = "https://github.com/freelawproject/scorched.git"}
 djangorestframework-filters = "1.0.0.dev2"
 ipython = "^7.21.0"
 gunicorn = "^20.0.4"
-eyecite = "<2.0.0"
+eyecite = {git = "https://github.com/freelawproject/eyecite.git", rev = "0595c93"}
```

Comment:
Nice. Let's cut a release for this once your counting PR is merged?

Reply:
Actually, I think it would be easier/safer to do the counting refactor as another PR after this one. I'm also not sure how long it will take me to do the tests for the counting PR in eyecite. But we can also hold off here until that's finalized, it doesn't matter much to me.

Reply:
So long as somebody is thinking about this, I'm happy.

Reply:
OK, I think this is the last piece that's missing before we merge this, right? Otherwise, we're good to go here as far as I can tell.

Reply:
Yeah, what did we decide here? Do you want to deploy a versioned tag of eyecite versus one just referencing a commit?